axon-framework/queries/query-dispatchers.md

```diff
@@ -227,5 +227,132 @@ commandGateway.sendAndWait(new RedeemCmd("gc1", amount));
 
    Since we subscribed to updates with the `println()` method, the update will be printed out once it is received.
 
+
+### Streaming queries
+
+The Streaming query allows a client to stream large database result sets. The Streaming query relies on reactive stream model, specifically the `Flux` type. Using streaming query, does not imply that reactive db drivers need to be used, users can hook to the stream and control the flow of data.
```

Comment:
I am not overly certain about the last sentence in this paragraph.
Other changes I made here, are consistently using "streaming" instead of "Streaming".

```suggestion
The streaming query allows a client to, for example, stream large database result sets. The streaming query relies on the reactive stream model, specifically the `Flux` type. Using the streaming query does not imply that reactive database drivers should be used. Users can hook to the stream and control the flow of data.
```